manifest_staging/charts/secrets-store-csi-driver/README.md

```diff
@@ -37,6 +37,7 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `linux.crds.image.repository`           | Linux crds image repository                                                                                                                                                    | `k8s.gcr.io/csi-secrets-store/driver-crds`              |
 | `linux.crds.image.pullPolicy`           | Linux crds image pull policy                                                                                                                                                   | `IfNotPresent`                                          |
 | `linux.crds.image.tag`                  | Linux crds image tag                                                                                                                                                           | `v1.2.4`                                                |
+| `linux.crds.resources`                  | The resource request/limits for the linux crds container images                                                                                                                | `{}`                                                    |
```

Comment:
```suggestion
| `linux.crds.resources`                  | The resource request/limits for the linux crds container image                                                                                                                | `{}`                                                    |
```

Reply:
I agree. I will change it.

Should I also change the chart version or will your team do that when you make a new release and move it to the other folder?

Reply:
The chart versions are upgraded during release.